build/subgraph.yaml

```diff
@@ -62,3 +62,63 @@ dataSources:
         - event: VoteCast(address,uint256,bool,uint256)
           handler: handleVoteCast
       file: GTCGovernorAlpha/GTCGovernorAlpha.wasm
+  - kind: ethereum/contract
+    name: PoolTogetherGovernorAlpha
+    network: mainnet
+    source:
+      address: "0xb3a87172f555ae2a2ab79be60b336d2f7d0187f0"
+      abi: PoolTogetherGovernorAlpha
+      startBlock: 11870114
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - User
+        - Proposal
+        - Vote
+      abis:
+        - name: PoolTogetherGovernorAlpha
+          file: PoolTogetherGovernorAlpha/abis/PoolTogetherGovernorAlpha.json
+      eventHandlers:
+        - event: ProposalCanceled(uint256)
+          handler: handleProposalCanceled
+        - event: ProposalCreated(uint256,address,address[],uint256[],string[],bytes[],uint256,uint256,string)
+          handler: handleProposalCreated
+        - event: ProposalExecuted(uint256)
+          handler: handleProposalExecuted
+        - event: ProposalQueued(uint256,uint256)
+          handler: handleProposalQueued
+        - event: VoteCast(address,uint256,bool,uint256)
+          handler: handleVoteCast
+      file: PoolTogetherGovernorAlpha/PoolTogetherGovernorAlpha.wasm
+  - kind: ethereum/contract
+    name: UniswapGovernorAlpha
+    network: mainnet
+    source:
+      address: "0x5e4be8bc9637f0eaa1a755019e06a68ce081d58f"
+      abi: UniswapGovernorAlpha
+      startBlock: 3402549
```

Comment:
Per this https://etherscan.io/tx/0x406c6b57eeecd07fd008f28ca67155bf115c54513e899305a0f2b96fb1637790, the contract was deployed at block 10861678. Why do you have 3402549?